models/gc_tiger_lb2/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,54 @@
+# Specify the base image for the environment
+FROM mhubai/base:latest
+
+# Specify/override authors label
+LABEL authors="sil.vandeleemput@radboudumc.nl"
+
+# install required dependencies for algorithm
+RUN pip3 install --no-cache-dir torch==2.0.1+cu118 torchvision==0.15.2+cu118 -f https://download.pytorch.org/whl/torch_stable.html
+
+# Install ASAP
+RUN apt-get update \
+    && apt-get -y install curl libpython3.8-dev \
+    && curl --remote-name --location "https://github.com/computationalpathologygroup/ASAP/releases/download/ASAP-2.1/ASAP-2.1-py38-Ubuntu2004.deb" \
+    && dpkg --install ASAP-2.1-py38-Ubuntu2004.deb || true \
+    && apt-get -f install --fix-missing --fix-broken --assume-yes \
+    && ldconfig -v \
+    && apt-get clean \
+    && echo "/opt/ASAP/bin" > /usr/local/lib/python3.8/dist-packages/asap.pth \
+    && rm ASAP-2.1-py38-Ubuntu2004.deb
+
+# Install tiger LB2 algorithm
+#   - Clone tiger LB2 codebase (master branch, fixed to commit 720f8dfca4624792c8e57915c4222efec5a0c2d4)
+#   - Subsequently we remove the .git directory to procuce a compacter docker layer
+RUN git clone https://github.com/vuno/tiger_challenge.git /vuno && \
+    cd /vuno && git reset --hard 720f8dfca4624792c8e57915c4222efec5a0c2d4 && \
+    rm -rf /vuno/.git
+
+# Install tiger LB2 dependencies
+RUN pip3 install --no-cache-dir -r /vuno/requirements.txt
+
+# Reinstall correct version of Numpy to function with ASAP 2.1
+RUN pip3 install --no-cache-dir --force-reinstall numpy==1.22
```

Comment:
This might clash with future mhubio updates. One option as once discussed could be wrapping up all algorithm dependencies in a pip env and then calling the cli script in that context.

Reply:
Could be, but the modules used for this model don't use Numpy as far as I can tell correct? I rather not touch this, since this will be very hard to configure correctly in combination with ASAP.

Reply:
Will you go back and fix it the moment we use Numpy in e.g. calculate general run-time statistics :)?
I already made a suggestion, you can install ASAP and the fixed Numpy version in a pip-env.

Please update before we can proceed.

Reply:
I'll try to install them in a pip-env, it'll give some overhead, but less maintainability issues.

Reply:
It has been installed inside a pipenv.